### ISSUE TITLE
Triggered by EBV infection

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6232]
**Preferred term label:**
Triggered by EBV infection

**Synonyms**
Triggered by Epstein Barr virus infection

**Definition (free text, please give PubMed ID)**
Applies to a sign or symptom that is provoked or brought about by infection by the Epstein-Barr virus.
 PMID:19425169 

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0025204

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://omim.org/entry/613011

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
done
